soccer/Robot.cpp

```diff
@@ -300,7 +305,9 @@ void OurRobot::unkick() {
     *_cmdText << "unkick()" << endl;
 }
 
-void OurRobot::kickImmediately(bool im) { radioTx.set_kick_immediate(im); }
+void OurRobot::kickImmediately(bool im) {
```

Comment:
Since we're not using the bool parameter anymore, we should probably get rid of it